### DISCUSSION TITLE
compatible heat-break

### DISCUSSION THREAD

**Discussion by ercintorun** [bassamanator/everything-sovol-sv06#28] [General]
https://tr.aliexpress.com/item/1005002679989261.html?spm=a2g0o.productlist.main.27.3ccc52f3uq5Omz&algo_pvid=3c2656f3-55e6-404b-8e48-dd484aaf9e2c&algo_exp_id=3c2656f3-55e6-404b-8e48-dd484aaf9e2c-13&pdp_npi=4%40dis%21TRY%2187.19%2161.15%21%21%213.08%21%21%402101d64d16937372457026931eee1a%2112000025807719435%21sea%21TR%210%21A&curPageLogUid=322IeK153vAI 

3dsway TA-C Thread   dimensions are the same, you can add to the readmes.

**Comment by ercintorun** [comment]
sorry i realised it is threaded, not compatible

**Comment by bassamanator** [comment]
I'm always on the lookout for replacements parts as well. Thanks for looking out. 

Inspired my your post, I created a discussion where users can list their findings as to potentially compatible parts, under General. Thanks again.
